sycl/unittests/kernel-and-program/Cache.cpp

```diff
@@ -1,568 +0,0 @@
-//==--------- Cache.cpp --- kernel and program cache unit test -------------==//
```

Comment:
I think that we don't won't to completely lose those tests for cache mechanism. I'm not suggesting to rewrite them in scope of this PR, but we should at least submit an issue about rewriting them to kernel bundles

Reply:
Okay, I've disabled these tests in 5056a12b9b47343502036f670e2e2b24fff34f37 instead of removing.

Reply:
> Okay, I've disabled these tests in https://github.com/intel/llvm/commit/5056a12b9b47343502036f670e2e2b24fff34f37 instead of removing.

Note here: I'm ok if we remove those tests for now, but we should submit an issue about returning them back in an updated form. I also think that not only `Cache` test should be eventually brought back, but probably a few others as well which are being removed in this PR.

I will let someone from @intel/llvm-reviewers-runtime to comment about what is the best/right approach here, because I'm not a code owner of the component